examples/browser/package.json

```diff
@@ -39,6 +40,8 @@
   "devDependencies": {
     "@types/chai": "^3.5.1",
     "@types/webdriverio": "^4.7.0",
+    "@types/yargs": "^6.6.0",
+    "bunyan": "^1.8.10",
```

Comment:
Actually this one is to be able to run:
npm start::backend as ```node ./lib/backend/main.js | bunyan```
To it enables piping the output to the buyan executable